spec/index.bs

```diff
@@ -5013,14 +5014,19 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
         `double` which is converted to the WGSL data type of the corresponding pipeline-overridable
         constant (`bool`, `i32`, `u32`, or `f32`) via [=converted to an IDL value|an IDL value=]
         ({{boolean}}, {{long}}, {{unsigned long}}, or {{float}}).
+        If the {{GPUFeatureName/"fp16-in-shader-and-buffer"}} [=feature=] is enabled, a
+        pipeline-overridable constant may be of WGSL data type `f16`. The specified
+        {{GPUPipelineConstantValue}} for a pipeline-overridable constant of `f16` is convert to
+        {{float}} via [=converted to an IDL value|an IDL value=], then convert to WGSL type `f16`
+        via IEEE-754 binary32 to IEEE-754 binary16 conversion.
```

Comment:
Is this strictly defined? In particular for rounding, truncation, and overflow? If so, can we link to an appropriate spec?

Reply:
WGSL spec defines the corresponding conversion, as in https://www.w3.org/TR/WGSL/#floating-point-conversion
Maybe we can say:
> The specified GPUPipelineConstantValue for a pipeline-overridable constant of `f16` is convert to float via an IDL value, then convert to WGSL type `f16` following the WGSL `f32` to `f16` conversion defined in [WGSL Spec](https://www.w3.org/TR/WGSL/#floating-point-conversion).

Reply:
The floating point conversion section in WGSL spec are also rephrased in this PR to explicitly spell out `f16` types, and indicate `f32` to `f16` conversion should follow the algorithm defined in this section.